### ISSUE TITLE
Published WCAG2ICT Note SOTD says this is a draft document - which it is not

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#634]
For some reason, when the WCAG2ICT Note was published, the SOTD wasn't updated to indicate it was a completed Note. It currently says:
>This is a draft document and may be updated, replaced or obsoleted by other documents at any time. It is inappropriate to cite this document as other than work in progress.

Be sure that the next time this is published that the status of the document is reflected correctly.

**Comment by daniel-montalvo** [comment]
Thanks for flagging this, @maryjom 

This is [required wording from W3C publication rules](https://www.w3.org/pubrules/doc/rules/?profile=NOTE#draftStability).

I concur this is misleading and should be given a thought as to how to further distinguish draft documents from more stable documents, irrespective of whether they are in NOTE or REC Track.

Are you happy for me to transfer this issue to the relevant repo in w3c?

**Comment by maryjom** [comment]
@daniel-montalvo sure, transfer it to where it is most appropriate. This language in the 2013 published and completed Working Group Note had this text, which I assume was not part of the template but instead added and edited by our TF:

>As a Working Group Note this content is stable, and the Working Group does not plan to make further changes. Should the need arise, however, the document could be updated. Comments received on this document will help the Working Group to decide if updates are needed, or will be taken into account should a republication be planned. Please send any comments on the “Additional Guidance” sections of this document to the public mailing list [public-wcag2ict-comments@w3.org](mailto:public-wcag2ict-comments@w3.org). Please include the following in your comments: the title of the document, location within the document, the concern, the suggested change, and any additional rationale for your comment.
